mw-webapp/cypress/e2e/navigationMenu.cy.ts

```diff
@@ -53,5 +54,30 @@ describe('Navigation menu scope tests', () => {
         cy.url().should('include', aboutProjectPageData.endpoint);
         aboutProjectSelectors.welcomeBlock.getTitle().should('contain', aboutProjectPageContent.mainTitle.en);
     });
+
+    it('NoAuth_NavMenu_Close', () => {
+        cy.get('body').click(345, 10, {force: true}); 
+
+        cy.url().should('include', '/');
+        navigationMenuSelectors.menuItemLinks.getAboutProjectItemLink().should('not.exist');
+    });
+
+    it('NoAuth_NavMenu_Home_Language', () => {
+        const languageCode = Object.keys(sideBarContent.language).map(key => key.toUpperCase());
+        const languageText = Object.values(sideBarContent.language);
+
+        navigationMenuSelectors.language.getSelect().click();
+
+        languageCode.forEach((code, index) => {
+            navigationMenuSelectors.language.languageMenuItems[index]().should('have.text', code);
+            navigationMenuSelectors.language.languageMenuItems[index]().click();
+    
+            navigationMenuSelectors.language.getText().should('have.text', languageText[index]);
+
+            if (index < languageText.length - 1) {
+                navigationMenuSelectors.language.getSelect().click();
+              }
+        });
```

Comment:
why need to click if this test check languages? Maybe I don't understand something, then ok(

Reply:
I switch languages ​​sequentially and check that the language changes in the selector and menu.